manifests/custom_service.pp

```diff
@@ -54,7 +55,7 @@
   # Remove legacy files so that we don't end up with conflicts
   $_safe_filename = firewalld::safe_filename($filename)
 
-  unless $_safe_filename == $short {
+  unless $_safe_filename == $name {
     file { "${config_dir}/${_safe_filename}.xml":
```

Comment:
Is this resource needed anymore?

Reply:
Yeah, it's how the old stuff gets cleaned up that may have an incompatible name.